spacy_llm/models/rest/openai/registry.py

```diff
@@ -43,6 +67,45 @@ def openai_gpt_4(
     )
 
 
+@registry.llm_models("spacy.GPT-3-5.v2")
+def openai_gpt_3_5_v2(
+    config: Dict[Any, Any] = SimpleFrozenDict(temperature=_DEFAULT_TEMPERATURE),
+    name: Literal[
+        "gpt-3.5-turbo",
+        "gpt-3.5-turbo-16k",
+        "gpt-3.5-turbo-0613",
+        "gpt-3.5-turbo-0613-16k",
+    ] = "gpt-3.5-turbo",  # noqa: F722,F821
+    strict: bool = OpenAI.DEFAULT_STRICT,
+    max_tries: int = OpenAI.DEFAULT_MAX_TRIES,
+    interval: float = OpenAI.DEFAULT_INTERVAL,
+    max_request_time: float = OpenAI.DEFAULT_MAX_REQUEST_TIME,
+) -> Callable[[Iterable[str]], Iterable[str]]:
+    """Returns OpenAI instance for 'gpt-3.5' model using REST to prompt API.
+    config (Dict[Any, Any]): LLM config arguments passed on to the initialization of the model instance.
+    name (Literal["gpt-3.5-turbo", "gpt-3.5-turbo-16k", "gpt-3.5-turbo-0613", "gpt-3.5-turbo-0613-16k"]): Model to use.
+    strict (bool): If True, ValueError is raised if the LLM API returns a malformed response (i. e. any kind of JSON
+        or other response object that does not conform to the expectation of how a well-formed response object from
+        this API should look like). If False, the API error responses are returned by __call__(), but no error will
+        be raised.
+    max_tries (int): Max. number of tries for API request.
+    interval (float): Time interval (in seconds) for API retries in seconds. We implement a base 2 exponential backoff
+        at each retry.
+    max_request_time (float): Max. time (in seconds) to wait for request to terminate before raising an exception.
+    RETURNS (Callable[[Iterable[str]], Iterable[str]]]): OpenAI instance for 'gpt-3.5' model using REST to
+        prompt API.
+    """
```

Comment:
Do we want to remove the docstrings and maybe just describe the params once at the top of the file? I feel like this is just a lot of bloat.

Reply:
I agree, I wanted to do something similar as well, just wasn't sure about the format.

Reply:
An argument against that would be if users would like to call these functions directly in Python code.

Reply:
Yeah, it's not great either way :thinking:

Reply:
https://github.com/explosion/spacy-llm/pull/236/commits/21bef95b16ba8747eea0b6563e8161362d578f68

Reply:
Good with me.